qa/workunits/ceph-helpers.sh

```diff
@@ -1742,7 +1742,19 @@ function run_tests() {
     export PATH=${CEPH_BUILD_VIRTUALENV}/ceph-disk-virtualenv/bin:${CEPH_BUILD_VIRTUALENV}/ceph-detect-init-virtualenv/bin:.:$PATH # make sure program from sources are preferred
     #export PATH=$CEPH_ROOT/src/ceph-disk/virtualenv/bin:$CEPH_ROOT/src/ceph-detect-init/virtualenv/bin:.:$PATH # make sure program from sources are preferred
 
-    export CEPH_MON="127.0.0.1:7109" # git grep '\<7109\>' : there must be only one
+    # Grab a random port from 7109 - 7200 that is not in use
+    # This should maybe be reworked
```

Comment:
It can't be random or else we will get spurious failures when doing ctest -j16 (or whatever). 

Maybe use a fixed port, unless you specify a 'batch' or something?